specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/modelManagement.json

```diff
@@ -0,0 +1,4755 @@
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "workspace",
+            "in": "path",
+            "description": "The name of the workspace.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Asset"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ModelErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Asset"
+        ],
+        "summary": "Delete an Asset.",
+        "description": "Delete the specified Asset.",
+        "operationId": "Assets_Delete",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "The Id of the Asset to delete.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "version",
+            "in": "path",
+            "description": "The version of the Microsoft.MachineLearning resource provider API to use.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "subscriptionId",
+            "in": "path",
+            "description": "The Azure subscription Id.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "resourceGroup",
+            "in": "path",
+            "description": "The name of resource group in which the workspace is located.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "workspace",
+            "in": "path",
+            "description": "The name of the workspace.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The resource exists and was deleted successfully."
+          },
+          "204": {
+            "description": "The resource does not exist and the request was well formed."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ModelErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Asset"
+        ],
+        "summary": "Update an Asset.",
+        "description": "Patch a specific Asset.",
+        "operationId": "Assets_Patch",
+        "consumes": [
+          "application/json-patch+json",
```

Comment:
Does this really accept json-patch+json?

Reply:
not really. fixed